### PR TITLE
4.0.1 updates

### PR DESCRIPTION
**Pull request by andrew-vancil**
- Tidied up old messy code and removed unnecessary manipulation
- Ensured that previous years are included in the summary
- Updated 'readr' and renv to address parsing errors

### CODE REVIEW

**Review by cole-brokamp — CHANGES_REQUESTED** [review]
- let's keep the `renv/settings.dcf` file, this is important for renv to be able to be portable
- keep all R code that the container is running inside of `entrypoint.R`
- make sure to add on the version of the container that is being used to the output filename; if you want to do this automatically, we could use something similar to `dht::write_geomarker_file()`, but might not be necessary here

File: weekly_data_report.R [hunk 1]
```diff
@@ -120,12 +88,12 @@ screen_neighborhood <- d_neigh %>%
 ##
 
 d_csv <- screen_neighborhood %>%
-  select(`Neighborhood` = neighborhood,
-         `Year` = year,
-         `Week` = week,
-         `Number of Calls` = n_calls,
-         `Number of Calls Screened In` = n_screened_in,
-         `Screen-In Rate` = screen_in_rate)
+  select(`neighborhood` = neighborhood,
```
Comment (cole-brokamp) [inline weekly_data_report.R:91]: we don't need the backticks here if the names don't contain any spaces

File: weekly_data_report.R [hunk 2]
```diff
@@ -50,50 +50,18 @@ d <- read_csv(args$file_name,
...
 ###
 d <- dplyr::mutate(d, DECISION_DATE = dht::check_dates(DECISION_DATE))
 
-# rmarkdown::render(input = '/app/aggregate_data_report.rmd',
-#                   params = list(d = d),
-#                   envir = new.env())
-
-
 tract_to_neighborhood <- readRDS('/app/tract_to_neighborhood.rds')
 neighborhood_shp <- readRDS('/app/ham_neighborhoods_dep_index_shp.rds')
```
[the remark this answers is not in the document]
Comment (andrew-vancil) [reply weekly_data_report.R:59]: Yep, got rid of the neighborhood_shp and moved the other to the newly restored entrypoint.r

**Comment by andrew-vancil** [comment]
Just tested after these changes and we're still good with previous years still present

**Review by cole-brokamp — APPROVED** [review]
thanks!  looking good... last thing:  can you keep tracking the test output file in git?  its good to be able to see the expected output when someone is trying to test or wants to look at an example input and output file

**Comment by andrew-vancil** [comment]
Gotcha, its back in there

**Pull request merged**